### PR TITLE
[REVIEW] Updated the cuML notebooks that were failing

### PR DESCRIPTION
**Pull request by Salonijain27**

### CODE REVIEW

File: cuml/umap_supervised_demo.ipynb [hunk 1]
```diff
@@ -204,7 +204,7 @@
     "record_data = (('fea%d'%i, data[:,i]) for i in range(data.shape[1]))\n",
     "gdf = cudf.DataFrame(record_data)\n",
     "\n",
-    "label_data = [('fea%0', target)]\n",
+    "label_data = np.asarray([('fea%0', target)])\n",
```
Comment (JohnZed) [inline cuml/umap_supervised_demo.ipynb]: Hmm... what is this array? It seems a little  odd that it's an np.array with a string with %0 in the name and then a vector of numbers maybe? Would be good to try rerunning this one.

Comment (Salonijain27) [reply cuml/umap_supervised_demo.ipynb]: updated it and changed the way cudf DataFrame was being created to match the cudf updates

File: cuml/umap_supervised_demo.ipynb [hunk 2]
```diff
@@ -203,10 +196,15 @@
     "%%time\n",
     "record_data = (('fea%d'%i, data[:,i]) for i in range(data.shape[1]))\n",
     "gdf = cudf.DataFrame(record_data)\n",
-    "\n",
-    "label_data = [('fea%0', target)]\n",
-    "\n",
-    "target_gdf = cudf.DataFrame(label_data)"
+    "#label_data = pd.DataFrame({'fea%0': target})\n",
```
Comment (cjnolet) [inline cuml/umap_supervised_demo.ipynb]: Remove comment

File: cuml/umap_supervised_demo.ipynb [hunk 3]
```diff
@@ -203,10 +196,15 @@
...
+    "target_gdf = cudf.DataFrame({'fea0': target}) #, ('fea1', target)])"
```
Comment (cjnolet) [inline cuml/umap_supervised_demo.ipynb]: Remove comment

File: cuml/umap_supervised_demo.ipynb [hunk 4]
```diff
@@ -203,10 +196,15 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There is cost with moving data between host memory and device memory (GPU memory) and we will include that core when comparing speeds"
```
Comment (cjnolet) [inline cuml/umap_supervised_demo.ipynb]: Should this `and we will include that cost`?

**Comment by taureandyernv** [comment]
@cjnolet, let's remove or update umap in a next PR :)

**Pull request merged**